components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -83,6 +77,24 @@ class AppLinksUseCases(
             .toHashSet()
     }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun getBrowserPackageNames(): Set<String> {
+        if (browserPackageNames != null) {
+            return browserPackageNames
+        }
+
+        val currentTimeStamp = System.currentTimeMillis()
```

Comment:
Since this time can jump forward/backswards (timezones, NTP update, etc.) we may want to use `SystemClock.elapsedRealtime()`.

Reply:
Sure, I'll update.